fp.js

```diff
@@ -9,6 +9,18 @@ var people = [
 people.forEach(person => {
   peoplesAges.push(person.age)
 })
+var peoplesAges = []
+var people = [
+  {name: "Jane", age: 32},
+  {name: "Jessica", age: 34},
+  {name: "Jasmine", age: 39}
+]
+people.forEach(person => {
+  peoplesAges.push(person.age)
+})
+var peopleOrder = people.map(function(x){
+return x.name + x.age;
+});
```

Comment:
:+1: The resulting an array should only contain ages, but will give you credit for using `map` correctly!